Specify prerequisites for DBMS instance to be monitored (#43)
neo4j/docs-ops-manager: modules/ROOT/pages/addition/index.adoc

Adding a DBMS in NOM’s terminology means to enable the monitoring of every instance in the DBMS by installing a NOM agent on the instance host and configuring it appropriately.

[[add-single]]
== Add a single instance DBMS

> Follow the steps below to add the single DBMS instance

Reviewer: ```suggestion
Follow the steps below to add the single DBMS instance:
```

[[add-cluster]]
== Add a cluster

> Follow the steps below to add *every* instance in the cluster.

Reviewer: ```suggestion
Follow the steps below to add *every* instance in the cluster:
```

== Add a DBMS instance

[[prerequisites]]
=== Prerequisites

To be monitored by NOM an instance of a DBMS must: 

> * Be running a Neo4j Enterprise Edition v4.4.0 or higher

Reviewer: ```suggestion
* be running a Neo4j Enterprise Edition v4.4.0 or higher
```

> * Be running a Neo4j Enterprise Edition version that is 4.4.0 or higher

Reviewer: ```suggestion
* Be running a Neo4j Enterprise Edition v4.4.0 or higher
```

> * Have the following config settings set in neo4j.conf: 

Reviewer: ```suggestion
* have the following config settings set in neo4j.conf: 
```

> ** metrics.prometheus.endpoint=127.0.0.1:2004

Reviewer: ```suggestion
** `metrics.prometheus.endpoint=127.0.0.1:2004`
```

> ** metrics.prometheus.enabled=true

Reviewer: ```suggestion
** `metrics.prometheus.enabled=true`
```

> ** metrics.enabled=true

Reviewer: ```suggestion
** `metrics.enabled=true`
```

> ** metrics.filter=*

Reviewer: ```suggestion
** `metrics.filter=*`
```

> ** metrics.jmx.enabled=true

Reviewer: ```suggestion
** `metrics.jmx.enabled=true`
```

> ** metrics.namespaces.enabled=true

Reviewer: ```suggestion
** `metrics.namespaces.enabled=true`
```

[[register]]
=== Register agent

> Before adding a DBMS instance for NOM monitoring, an agent needs to be registered with NOM server which provides server communication configuration for the agent.

Reviewer: ```suggestion
Before adding a DBMS instance for NOM monitoring, one agent needs to be registered with the NOM server which provides server communication configuration for the agent.
```
